src/pybind/mgr/rbd_support/schedule.py

```diff
@@ -470,27 +520,30 @@ def remove(self, level_spec, interval, start_time):
                 del self.level_specs[level_spec.id]
         self.save(level_spec, schedule)
 
-    def find(self, pool_id, namespace, image_id=None):
-        levels = [None, pool_id, namespace]
+    def find(self,
+             pool_id: str,
+             namespace: str,
+             image_id: Optional[str] = None) -> Optional['Schedule']:
+        levels = [pool_id, namespace]
         if image_id:
             levels.append(image_id)
 
         while levels:
-            level_spec_id = "/".join(levels[1:])
+            level_spec_id = "/".join(levels)
```

Comment:
@tchaikov this breaks global schedule. Not sure why you didn't see this, because it breaks `qa/workunits/rbd/cli_generic.sh` for me.

The purpose of `None` in `levels` was to make the `while levels` loop to run for "global" case. I.e. when `levels == [None]`, then `"/".join(levels[1:]) == "/".join([]) == ""`. And with you change you just skip `level_spec_id = ""` case.

If you don't like `None` in the list, you can use an empty string. Anything will work there, as it is used only as a marker. I chose `None` to give a hint of its special purpose.